backend/LexBoxApi/GraphQL/GraphQlSetupKernel.cs

```diff
@@ -50,10 +50,6 @@ public static void AddLexGraphQL(this IServiceCollection services, IHostEnvironm
                 options.IncludeExceptionDetails = true;
             })
             .AddType<DbErrorCode>()
-            .AddType(new DateTimeType("DateTime"))
-            .AddType(new UuidType("UUID"))
-            .AddType(new DateTimeType("timestamptz"))
-            .AddType(new UuidType("uuid"))
```

Comment:
it turns out these are no longer needed. They were required back when we used Hasura and were just never removed